magefiles/main.go

```diff
@@ -218,7 +218,7 @@ func LocalDev(arg string) error {
 
 	switch arg {
 	case "minimal":
-		os.Setenv("ARMADA_COMPONENTS", "executor,server,scheduler")
+		os.Setenv("ARMADA_COMPONENTS", "executor,server,scheduler,lookoutingesterv2")
```

Comment:
do we need this?  I think in the docker-compose the server depend on the lookoutingester.  If it doesn't , then it should!